eng/testing/linker/SupportFiles/Directory.Build.targets

```diff
@@ -29,6 +31,18 @@
     </ItemGroup>
   </Target>
 
+  <Target Name="RootJavaScriptInteropAssembly"
+          BeforeTargets="_RunILLink"
```

Comment:
`_RunILLink` is a private target (since it has an underscore). What you really want is `BeforeTargets="PrepareForILLink"`